### DISCUSSION TITLE
netbox.netbox.nb_inventory plugin: How to group by two variables?

### DISCUSSION THREAD

**Discussion by markbzz** [netbox-community/ansible_modules#565] [Q&A]
ISSUE TYPE
 Module question

SOFTWARE VERSIONS
Ansible:
2.10.0

Netbox:
v2.11.6

Collection:
$ ansible-galaxy collection list | grep netbox
netbox.netbox 3.0.0
netbox.netbox 3.1.1

SUMMARY
I am able to pull Netbox inventory values by using the netbox.netbox.nb_inventory.
Using group_by, I am able to group by sites for the hostnames.

How do I group by the hostname and the ansible_host (IP address)?

**Comment by ryanmerolle** [comment]
Your question is sort of confusing, but I do not think your problem is the group_by function, but it sounds like you want to spit out devices as hostnames and you have another use case to spit out the IP?  Is that correct?
